pwndbg/disasm/arm.py

```diff
@@ -46,7 +46,7 @@ def condition(self, instruction):
         if instruction.address != pwndbg.regs.pc:
             return False
 
-        cpsr = pwndbg.regs.cpsr
+        cpsr = pwndbg.regs.cpsr or pwndbg.regs.xpsr
```

Comment:
if `cpsr` is 0 we will use xpsr, is it correct?

Reply:
err, right. same as above.